protocol/x/clob/keeper/clob_pair.go

```diff
@@ -165,19 +168,6 @@ func (k Keeper) createOrderbook(ctx sdk.Context, clobPair types.ClobPair) {
 // CreateClobPair creates a new `ClobPair` in the store and creates the corresponding orderbook in the memclob.
 // This function returns an error if a value for the ClobPair's id already exists in state.
 func (k Keeper) CreateClobPair(ctx sdk.Context, clobPair types.ClobPair) error {
-	// Validate the given clob pair id is not already in use.
-	if _, exists := k.GetClobPair(ctx, clobPair.GetClobPairId()); exists {
```

Comment:
is removal of this safe? does this change anything existing?

Reply:
This validation already exists in the validate function here: https://github.com/dydxprotocol/v4-chain/blob/d40ab1119f059029318484dde6f6b74871d42c69/protocol/x/clob/keeper/clob_pair.go#L84